2nd-gen/packages/swc/components/asset/stories/asset.stories.ts

```diff
@@ -137,33 +161,51 @@ export const Folder: Story = {
 //    ACCESSIBILITY STORIES
 // ────────────────────────────────
 
-const accessibilityArgs = [
-    {
-        variant: 'folder',
-        label: '/packages/swc/ folder',
-    },
-    {
-        label: 'Sara Sawyer avatar',
-        'default-slot': `<img src="https://picsum.photos/id/823/80/80" alt="preview of the user profile picture" />`,
-    },
-];
 /**
  * ### Features
  *
- * The `<sp-asset>` element implements several accessibility features:
+ * The `<swc-asset>` element implements several accessibility features:
  *
- * 1. **Labeling**: Uses the `label` attribute value as `aria-label`
+ * #### ARIA implementation
  *
- * ### Best Practices
+ * - **Icon labeling**: File and folder SVG icons automatically use the `label` property as `aria-label`
+ * - **Semantic role**: SVG icons use `role="img"` to identify them as image content
```

Comment:
This is not provided by the component itself but by the slotted content. TBH it would be worth exploring setting the role on asset, but that's not what it does today.

```suggestion
```